codegen-plugin/src/main/kotlin/io/spine/protodata/hello/BuilderExtensionGenerator.kt

```diff
@@ -86,6 +93,11 @@ internal class BuilderExtensionGenerator(
             val functionName = "validate" + fieldName.camelCase() + "Count"
             val builderClass = fullClassName.nestedClass("Builder")
 
+            val validationMethod = "$javaPackage.$simpleTypeName" +
+                    "BuilderExtKt.$functionName(this);"
```

Comment:
Also, the file naming scheme we usually go for extensions is `...Exts.kt`.

See [this part](https://github.com/SpineEventEngine/documentation/wiki/Kotlin-code-style#extension-functions-and-properties) of Spine's developer documentation.